src/sass/file-preview-card/file-preview-card.scss

```diff
@@ -0,0 +1,164 @@
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.file-preview-card {
+    @include background-color-token(
+        file-preview-card-background-color,
+        color-background-secondary
+    );
+
+    @include border-radius-token(
+        file-preview-card-border-radius,
+        border-radius-100
+    );
+
+    // Safari support in v15+ reviewed with team
+    aspect-ratio: 1/1;
+    box-sizing: border-box;
+    display: grid;
+    grid-template-areas: " . . menu-action " " . center-action . " " label . . ";
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: 100%;
+    max-width: 396px;
+    min-width: 120px;
+    width: 100%;
+}
+
+.file-preview-card__body {
+    align-items: center;
+    display: flex;
+    grid-area: 1 / 1 / 4 / 4;
+    justify-content: center;
+}
+
+.file-preview-card__body img,
+.file-preview-card__body video {
+    @include border-radius-token(
+        file-preview-card-border-radius,
+        border-radius-100
+    );
+
+    height: 100%;
+    object-fit: contain;
+    width: 100%;
+}
+
+.file-preview-card__body--see-more {
```

Comment:
Not a huge deal, but if the only style change here is opacity,  I wonder if this should be named something like `.file-preview-card__body--fade` in order to have a more generic use?